tests/models/test_cpu.py

```diff
@@ -379,151 +378,3 @@ def train_dataloader(self):
     result = trainer.fit(model)
 
     assert result == 1, 'training failed to complete'
-
-
-def test_tbptt_cpu_model_result(tmpdir):
```

Comment:
Why are these gone?

Reply:
do not ask me :D you try git blame who removed/rewrote relevant tests :]